transport/packet.go

```diff
@@ -26,11 +26,11 @@ type PacketEndpoint interface {
 	Connect(ctx context.Context) (net.Conn, error)
 }
 
-// UDPEndpoint is a [PacketEndpoint] that connects to the given address via UDP
+// UDPEndpoint is a [PacketEndpoint] that connects to the specified address using UDP
```

Comment:
[UDPEndpoint]

Reply:
Add a period at the end of the sentence.

Reply:
Type names should not be wrapped in `[]` in the documentation that describes the type itself. Added "." at the end.